12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,17 @@
-Glob Lab.
-https://www.tocode.co.il/bundles/unix1/lessons/globlab
-
-1. Type in a glob pattern to match:
-
-  1. All files whose name starts with a lowercase letter
-  Answer: 
-
-  2. All files whose name starts with an uppercase letter
-  Answer:
-
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
-  Answer:
-
-  4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
-  Answer:
-
-  5. All files with extension ".c" that are in a folder named "src"
-  Answer:
-
-  6. All files with extension ".c" under any folder
-  Answer:
-
-2. Write a command that would delete all files ending with a ".png" under any subdirectory.
-
-3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
-
-4. Search in "man ls" for an option to show only directory data so the command "ls _____ D*" should print only the word Doc
-
+1. Mkdir  wildcards
+2. Cd wildcards � mkdir� (all folder listed
+3. Touch command for files.
+4. 
+a. echo [a-z]* (using this command export LC_ALL=POSIX)
+b. echo [A-Z]*
+c. echo * [a,e,i,o,u]

```

Comment:
The space is an error, and commas are not required. Correct answer is: `echo *[aeiou]`

(comma is a normal character, and not required for separation. The space meant you'll get "All Files" AND "All files that are the single letter a,e,i,o or u", whereas what you wanted is the all files that end in a vower)

Spaces in shell are very important as they function as word separators.